### ISSUE TITLE
Empty button in Side Panel

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#4445]
## Issue Description

There is an empty button called "toggle navigation" that is not visible but is accessible programatically via the keyboard and assistive technology.

## WCAG Criteria

[1.3.1 Info and Relationships - Level A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Screen recording

In the screen recording you can see that the VoiceOver Cursor essentially dissapears because there is no visible element on the page to move its focus too. 

https://share.zight.com/DOu1KLez

## Code Snippet

```html
<div role="button" tabindex="0" class="nav-mask active" aria-label="Toggle navigation"></div>
```

## Notes

If it has no purpose it should be deleted. If there is a reason for it to be here it should be programatically hidden as long as it is visually hidden.

In user testing this button in conjunction with the visible toggle button for the sidepanel created some confusion.

**Comment by danny-avila** [comment]
This "button" is a drag-able element for the width of the side panel, what do you recommend for that?

**Comment by derekjackson-das** [comment]
Do we have the same element? In the demo site this div has no clickable area and if I give it some width and click on it it will expand and close the side panel. 

https://share.zight.com/o0u1jzPA

**Comment by danny-avila** [comment]
The right-hand side panel needs to be opened (the right-side arrow/carat will open it)

**Comment by derekjackson-das** [comment]
Hey @danny-avila, I am sorry I still don't see that draggable element on the left panel, even with the right panel open. I do see the draggable element in the right side panel but it is different than the empty button on the left panel. Is there supposed to be another on the left side?

This is the draggable control I see on the right panel:

`<div class="focus-visible:ring-ring group relative flex w-px items-center justify-center after:absolute after:inset-y-0 after:left-1/2 after:w-1 after:-translate-x-1/2 focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-offset-1 data-[panel-group-direction=vertical]:h-px data-[panel-group-direction=vertical]:w-full data-[panel-group-direction=vertical]:after:left-0 data-[panel-group-direction=vertical]:after:h-1 data-[panel-group-direction=vertical]:after:w-full data-[panel-group-direction=vertical]:after:-translate-y-1/2 data-[panel-group-direction=vertical]:after:translate-x-0 [&amp;[data-panel-group-direction=vertical]>div]:rotate-90 bg-transparent dark:text-white" role="separator" tabindex="0" data-panel-group-direction="horizontal" data-panel-group-id=":rl:" data-resize-handle="" data-resize-handle-state="inactive" data-panel-resize-handle-enabled="true" data-panel-resize-handle-id=":rv:" aria-controls="messages-view" aria-valuemax="80" aria-valuemin="60" aria-valuenow="97" style="touch-action: none; user-select: none;">`

and this is what I see on the left side panel:

`<div role="button" tabindex="0" class="nav-mask active" aria-label="Toggle navigation"></div>`

If it is easier I can jump on a quick call to see what I am missing or we can talk Friday?

**Comment by danny-avila** [comment]
This is the mobile navigation mask, that should not have been focus-able on larger screens. It's now properly removed on larger screens.
